bot/feedback/feedback.py

```diff
@@ -0,0 +1,303 @@
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            error (commands.CommandError): The error raised during the execution of the command.
+        """
+        if isinstance(error, commands.BadArgument):
+            await ctx.send("Ich konnte leider keinen Vorschlag mit der von dir angegebenen Nummer finden. :frowning2:",
+                           delete_after=constants.TIMEOUT_INFORMATION)
+
+    async def _change_suggestion_status(self, status: SuggestionStatus, author: discord.Member, guild: discord.Guild,
+                                        suggestion_id: int, reason: Optional[str]):
+        """Method which changes the status of a suggestion.
+
+        Changes the status of the suggestion in our db, adapts the corresponding embed in the suggestion channel to
+        represent the newly made changes and finally notifies the user who submitted it via DM.
+
+        Args:
+            status (SuggestionStatus): The new status of the suggestion.
+            author (discord.Member): The user who invoked the command to change it.
+            guild (discord.Guild): The Discord server on which to find the suggestion.
+            suggestion_id (int): The id of the suggestion.
+            reason (Optional[str]): The reason for the decision.
+        """
+        id_exists = self._db_connector.set_suggestion_status(suggestion_id, status)
+        if not id_exists:
+            raise commands.BadArgument
+
+        suggestion_data = self._db_connector.get_suggestion(suggestion_id)
+        message = await guild.get_channel(constants.CHANNEL_ID_SUGGESTIONS).fetch_message(suggestion_data[0])
+
+        await _refresh_suggestion_embed(message, author, reason, SuggestionStatus(suggestion_data[1]))
+        await _notify_user_suggestion_change(guild, int(suggestion_data[2]))
+
+    @commands.Cog.listener(name='on_raw_reaction_add')
+    async def suggestion_reaction_add(self, payload: discord.RawReactionActionEvent):
+        """Event listener which triggers if a reaction has been added by a user.
+
+        If the affected message is in the configured suggestion channel and the added reaction is one of the two vote
+        emojis specified in constants.py, the color of the suggestion embed will be changed if enough up- or downvotes
+        have been submitted.
+
+        Args:
+            payload (discord.RawReactionActionEvent): The payload for the triggered event.
+        """
+        if not payload.member.bot and payload.channel_id == constants.CHANNEL_ID_SUGGESTIONS and \
+                self._db_connector.get_suggestion_status(payload.message_id) == SuggestionStatus.UNDECIDED:
+            message = await self.bot.get_channel(payload.channel_id).fetch_message(payload.message_id)
+            reactions = message.reactions
+
+            if payload.emoji.name in (constants.EMOJI_UPVOTE, constants.EMOJI_DOWNVOTE):
+                total_votes = reactions[0].count + reactions[1].count
```

Comment:
You could directly use
`required_difference= (reactions[0].count + reactions[1].count) /2`